### PR TITLE
Issue 291: Remove section colours

### PR DESCRIPTION
**Pull request by christinem**
(Because I was on a roll.)
This fixes Issue #291. All sections now have the same colour (I just chose the colour for Lectures: blue3, unless anyone is opposed to this). 

This PR is built upon #312. That PR should be merged before this is considered.

### CODE REVIEW

File: hs/CssGen.hs [hunk 1]
```diff
@@ -591,9 +591,9 @@ tdColours = ".timetable " ?  do
     td # ("in-conflict" *= "false") # ("satisfied" *= "true") # ("type" *= "L") ? do
             backgroundColor blue3
     td # ("in-conflict" *= "false") # ("satisfied" *= "true") # ("type" *= "T") ? do
-            backgroundColor teal1
```
Comment (david-yz-liu) [inline hs/CssGen.hs]: Rather than repeat the selectors, why don't we just remove the `# ("type *= "L")`.

Comment (christinem) [inline hs/CssGen.hs]: I actually tried that at first but it just made all the lines on the grid blue :/

Comment (christinem) [inline hs/CssGen.hs]: Actually I could probably do it based on whether the clicked attribute is true or false. I'll make that change.

**Comment by david-yz-liu** [comment]
Ah, interesting. This is a result of a lack of css knowledge when the styling was first done (my fault). You should add a new selector under td to match `td:empty` and set the background to be white in this case.

**Comment by christinem** [comment]
@david-yz-liu Ah, alright, I can do that, thanks!

**Comment by christinem** [comment]
@david-yz-liu Hmm, adding a td:empty selector doesn't seem to solve the problem as well. Seems as though the ".timetable  td[in-conflict*='false'][satisfied*='true']" selector is overriding it, since it's more specific.

However, adding a [clicked*='true'] (as I was originally thinking) to the original selector and taking out the type produces the desired effect.

**Comment by david-yz-liu** [comment]
I'm not happy with adding more and more attributes in the selectors. You can nest an `:empty` inside the existing selector on line 591.

You also need to understand how nesting selectors works in Clay - you created a new selector, but this should be nested inside the td selector.

**Comment by christinem** [comment]
@david-yz-liu thanks, I think I've fixed it through using ```:empty```

**Comment by david-yz-liu** [comment]
Thanks!

**Pull request merged**